tests/e2e/configurer/chain/commands.go

```diff
@@ -267,6 +267,14 @@ func (n *NodeConfig) SubmitSuperfluidProposal(asset string, initialDeposit sdk.C
 	n.LogActionF("successfully submitted superfluid proposal for asset %s", asset)
 }
 
+func (n *NodeConfig) SubmitCreateConcentratedPoolProposal(initialDeposit sdk.Coin) {
+	n.LogActionF("Cretaing concentrated liquidity pool")
+	cmd := []string{"osmosisd", "tx", "gov", "submit-proposal", "create-concentratedliquidity-pool-proposal", "uosmo", "stake", "100", "0.01", "--title=\"create concentrated pool\"", "--description=\"create concentrated pool", "--from=val", fmt.Sprintf("--deposit=%s", initialDeposit)}
+	_, _, err := n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
```

Comment:
e2e testing failing here, does anyone know why this might be failing? 🤔

Reply:
Sounds like it fails to execute the CLI command.

You can try doing that against a running container manually and see what you get:
```
docker exec -it <container name> osmosisd ...
```

If this doesn't help, please let me know